src/code/z_message_PAL.c

```diff
@@ -114,7 +114,8 @@ void Message_ResetOcarinaNoteState(void) {
     R_OCARINA_NOTES_YPOS(2) = 179;
     R_OCARINA_NOTES_YPOS(3) = 174;
     R_OCARINA_NOTES_YPOS(4) = 169;
-    sOcarinaNoteBuf[0] = 0xFF;
+    sOcarinaNoteBuf[0] = OCARINA_NOTE_INVALID;
```

Comment:
this is being handled in #1026 as well as the comment on `sLastPlayedSong` and changes in `code_800EC960.c`

Reply:
Reverted.